app/assets/css/emojione-awesome.css

```diff
@@ -1,5500 +1,39 @@
-
-.e1a-triangular_ruler {
-  background-image: url("/images/emojione/svg/1f4d0.svg"); }
-
-.e1a-straight_ruler {
-  background-image: url("/images/emojione/svg/1f4cf.svg"); }
-
-.e1a-pushpin {
-  background-image: url("/images/emojione/svg/1f4cc.svg"); }
-
-.e1a-round_pushpin {
-  background-image: url("/images/emojione/svg/1f4cd.svg"); }
-
-.e1a-triangular_flag_on_post {
-  background-image: url("/images/emojione/svg/1f6a9.svg"); }
-
-.e1a-flag_white {
-  background-image: url("/images/emojione/svg/1f3f3.svg"); }
-
-.e1a-flag_black {
-  background-image: url("/images/emojione/svg/1f3f4.svg"); }
-
-.e1a-closed_lock_with_key {
-  background-image: url("/images/emojione/svg/1f510.svg"); }
-
-.e1a-lock {
-  background-image: url("/images/emojione/svg/1f512.svg"); }
-
-.e1a-unlock {
-  background-image: url("/images/emojione/svg/1f513.svg"); }
-
-.e1a-lock_with_ink_pen {
-  background-image: url("/images/emojione/svg/1f50f.svg"); }
-
-.e1a-pen_ballpoint {
-  background-image: url("/images/emojione/svg/1f58a.svg"); }
-
-.e1a-pen_fountain {
-  background-image: url("/images/emojione/svg/1f58b.svg"); }
-
-.e1a-black_nib {
-  background-image: url("/images/emojione/svg/2712.svg"); }
-
-.e1a-pencil {
-  background-image: url("/images/emojione/svg/1f4dd.svg"); }
-
-.e1a-pencil2 {
-  background-image: url("/images/emojione/svg/270f.svg"); }
-
-.e1a-crayon {
-  background-image: url("/images/emojione/svg/1f58d.svg"); }
-
-.e1a-paintbrush {
-  background-image: url("/images/emojione/svg/1f58c.svg"); }
-
-.e1a-mag {
-  background-image: url("/images/emojione/svg/1f50d.svg"); }
-
-.e1a-mag_right {
-  background-image: url("/images/emojione/svg/1f50e.svg"); }
-
-.e1a-heart {
-  background-image: url("/images/emojione/svg/2764.svg"); }
-
-.e1a-yellow_heart {
-  background-image: url("/images/emojione/svg/1f49b.svg"); }
-
-.e1a-green_heart {
-  background-image: url("/images/emojione/svg/1f49a.svg"); }
-
-.e1a-blue_heart {
-  background-image: url("/images/emojione/svg/1f499.svg"); }
-
-.e1a-purple_heart {
-  background-image: url("/images/emojione/svg/1f49c.svg"); }
-
-.e1a-broken_heart {
-  background-image: url("/images/emojione/svg/1f494.svg"); }
-
-.e1a-heart_exclamation {
-  background-image: url("/images/emojione/svg/2763.svg"); }
-
-.e1a-two_hearts {
-  background-image: url("/images/emojione/svg/1f495.svg"); }
-
-.e1a-revolving_hearts {
-  background-image: url("/images/emojione/svg/1f49e.svg"); }
-
-.e1a-heartbeat {
-  background-image: url("/images/emojione/svg/1f493.svg"); }
-
-.e1a-heartpulse {
-  background-image: url("/images/emojione/svg/1f497.svg"); }
-
-.e1a-sparkling_heart {
-  background-image: url("/images/emojione/svg/1f496.svg"); }
-
-.e1a-cupid {
-  background-image: url("/images/emojione/svg/1f498.svg"); }
-
-.e1a-gift_heart {
-  background-image: url("/images/emojione/svg/1f49d.svg"); }
-
-.e1a-heart_decoration {
-  background-image: url("/images/emojione/svg/1f49f.svg"); }
-
-.e1a-peace {
-  background-im
```

Comment:
we cannot use public URLs in the DD, are all the images up-to-date in the repo (I see no image change), can we use the relative version below everywhere?